test/system/255-auto-update.bats

```diff
@@ -305,7 +305,7 @@ EOF
         fi
     done
 
-    # Only check the last service is started. Previous services should already actived.
+    # Only check the last service is started. Previous services should already activated.
```

Comment:
Neither of these make sense. Maybe "already **be**" activated"? And, I dislike the word "should": it's ambiguous, sometimes meaning "we expect that", and sometimes meaning "we will take care of it". How about:
```
# Check that only the last service is started; previous services have been activated elsewhere
```
(but even that is cringy. I don't have a good sense for what is being tested here).